assets/javascript/script.js

```diff
@@ -37,37 +37,46 @@ $(document).ready(function () {
             console.log(results);
 
             for (var i = 0; i < results.length; i++) {
-
                 let recipeDiv = $("#dinner-table");
                 let recipeTitle = results[i].title;
                 let recipeID = results[i].id;
+                let SITE_BASE = "https://spoonacular.com/recipes/";
+
+                let recipeLink = $("<a>");
+                recipeLink.attr({
+                    href: SITE_BASE + recipeTitle + "-" + recipeID,
+                    target: "_blank",
+                })
+
                 let titleText = $("<h4>").text(recipeTitle);
-                let recipePrefix = "https://spoonacular.com/recipes/";
-            
+
                 let recipeImage = $("<img>");
                 recipeImage.attr({
                     class: "img-fluid img-thumbnail shadow rounded",
                     src: results[i].image,
                     value: recipeID,
                 })
-            
-                recipeDiv.prepend(titleText);
-                recipeDiv.prepend(recipeImage);
-            
+
+                recipeDiv.prepend(recipeLink)
```

Comment:
Why prepend? Won't this make the recipes show up in reverse order?

Reply:
The recipes are not in an "order".  It's entirely random and not in a particular order.  I used prepend only because Nate wanted them put at the top so prepend was the obvious choice.